src/main.rs

```diff
@@ -10,10 +10,8 @@ const CONCURRENT_TASKS: usize = 32768;
 static SEMA: Semaphore = Semaphore::const_new(CONCURRENT_TASKS);
 
 #[derive(Parser, Debug)]
-#[clap(rename_all = "kebab-case")]
```

Comment:
I'm curious why this is removed, is it a default behavior?

Reply:
Yeap https://github.com/cr0sh/jaso/pull/5/commits/6de48769c8622b7553830008830bf5b1d5348525

###### Reference
- https://docs.rs/clap/latest/clap/_derive/index.html#command-attributes